internal/service/resources/api/openapi.yaml

```diff
@@ -758,7 +758,6 @@ components:
             country: "ES"
       required:
       - oCloudId
-      - globalCloudId
```

Comment:
The specification says that this field is mandatory, not nullable, shouldn't be make it mandatory in the flags as well?

Reply:
actually since the type is a uuid the best we could do here to allow it to be null or to return a zero value uuid object since we can't force it to be an "".

Reply:
I've made it mandatory and left it as a zero uuid.